channels_fields/migrations/0004_field_channel_refactor.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 2.2.27 on 2022-07-21 19:05
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import imagekit.models.fields
+import profiles.utils
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("channels_fields", "0003_subfield"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name="fieldchannel",
+            name="avatar_medium",
+        ),
+        migrations.RemoveField(
+            model_name="fieldchannel",
+            name="avatar_small",
+        ),
+        migrations.RemoveField(
+            model_name="fieldchannel",
+            name="description",
+        ),
+        migrations.AddField(
+            model_name="fieldchannel",
+            name="public_description",
+            field=models.CharField(default="", max_length=80),
+        ),
+        migrations.AlterField(
+            model_name="fieldchannel",
+            name="avatar",
+            field=imagekit.models.fields.ProcessedImageField(
+                max_length=2083, null=True, upload_to=profiles.utils.avatar_uri
+            ),
+        ),
+        migrations.AlterField(
+            model_name="fieldchannel",
+            name="banner",
+            field=imagekit.models.fields.ProcessedImageField(
+                max_length=2083, null=True, upload_to=profiles.utils.banner_uri
+            ),
+        ),
+        migrations.AlterField(
+            model_name="fieldchannel",
+            name="name",
+            field=models.CharField(
+                max_length=100,
+                validators=[
```

Comment:
**Question:** Does this append a new validator to some list of validators for the field, or does it overwrite the list of validators for the field?

In particular, migration 0001 says `name` should be unique, but I was able to create two fields with the same name, and I'm wondering if maybe this is why?.

Reply:
Good catch, the `discussions.models.Channel` model doesn't have `unique=True` for the `name` attribute but I think it should, will change that and update the migration.